src/main/java/org/opentripplanner/raptor/rangeraptor/path/PathParetoSetComparators.java

```diff
@@ -26,19 +26,13 @@ private PathParetoSetComparators() {}
   public static <
     T extends RaptorTripSchedule
   > ParetoComparator<RaptorPath<T>> comparatorStandard() {
-    return (l, r) ->
-      l.endTime() < r.endTime() ||
-      l.numberOfTransfers() < r.numberOfTransfers() ||
-      l.durationInSeconds() < r.durationInSeconds();
```

Comment:
Is it intentional that you no longer compare the `durationInSeconds`?

Reply:
Yes. I did a little cleanup, but plan to change it even more later. The point is that the path should use the same criteria, as each stop-arrival. It does not matter for the heuristics, but it makes unit-testing much easier. There are som comments about this in some of the module tests where we do not get the expected path, due to this.  From commit message:

During the RangeRaptor Standard search the criteria is num-of-transfers and
arrival-time(main-criteria), but not duration. The timetable parameter can
still be used with the standard profiles.